iati/core/tests/test_utilities.py

```diff
@@ -191,3 +192,14 @@ def test_log_exception(self):
     def test_log_warning(self):
         """TODO: Implement testing for logging."""
         pass
+
+
+class TestDefaultVersions(object):
+    """A container for tests relating to default versions."""
+
+    @pytest.mark.parametrize("integer", iati.core.constants.STANDARD_VERSIONS_MAJOR)
```

Comment:
Suggest changing from `integer` to something like `major_version` to be clearer that it relates to a version number rather than a datatype.

(might be other cases like this)